js/controller/ContentController.js

```diff
@@ -18,14 +17,36 @@ app.controller('ContentController', function (Publisher, FeedResource, ItemResou
     // distribute data to models based on key
     Publisher.publishAll(data);
 
+    var getOrdering = function () {
```

Comment:
This function is unchanged, I just moved it up here since it is called by `getFirstItem` now.